frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/simulated-regions-modal/simulated-regions-modal.component.html

```diff
@@ -8,35 +8,50 @@ <h4 class="modal-title">Simulierte Regionen</h4>
     ></button>
 </div>
 <div
-    class="modal-body"
+    class="modal-body p-0"
     style="height: min(80vh, 2000px); max-height: min(80vh, 2000px)"
 >
-    <div class="h-100 d-flex flex-column">
-        <nav
-            #nav="ngbNav"
-            ngbNav
-            [(activeId)]="currentSimulatedRegionId"
-            class="nav-tabs"
+    <div class="d-flex flex-row w-100 h-100">
+        <div
+            class="flex-shrink-1 mh-100 p-4 d-flex flex-column"
+            style="min-width: 20%"
```

Comment:
There should also be a max width and I would suggest using units that do not depend on the screen size that much so that we know how much space we have when designing the components for the different radiograms